pkg/apis/rbac/types.go

```diff
@@ -115,6 +115,7 @@ type RoleBinding struct {
 
 	// RoleRef can reference a Role in the current namespace or a ClusterRole in the global namespace.
 	// If the RoleRef cannot be resolved, the Authorizer must return an error.
+	// This field is immutable.
```

Comment:
can you also change the comment on the ClusterRoleBinding type, and on the same types in the v1/types.go file? The versioned file is where API docs are generated from, so that will require running `make update` to regenerate API docs.

Reply:
Thanks for the feedback. I've added the comment to the ClusterRoleBinding and `v1/types.go` file and updated the API docs.